docs/simulators/DualSPHysics.md

```diff
@@ -48,6 +52,395 @@ task.download_outputs()
 
 ```
 
+Below, are going to show how to run a slighly more complex examples that come
```

Comment:
Below, we will demonstrate how to run a slightly more complex example included in the DualSPHysics distribution, located in the examples folder.

Reply:
Done.